pages/common/multipass.md

```diff
@@ -0,0 +1,36 @@
+# multipass
+
+> Creates a Ubuntu virtual machine using native hypervisors.
```

Comment:
Hi @monsieurlo what about?

```suggestion
> CLI to manage Ubuntu virtual machines using native hypervisors.
```

Reply:
Hi, I just pushed the modification. (Sorry for CLA thing, I completely forgot in the meantime I had changed my gitconfig locally :p )